test/jaas-test/src/main/java/com/griddynamics/jagger/test/jaas/invoker/ExecutionManipulateInvoker.java

```diff
@@ -0,0 +1,28 @@
+package com.griddynamics.jagger.test.jaas.invoker;
+
+import com.griddynamics.jagger.invoker.InvocationException;
+import com.griddynamics.jagger.invoker.v2.JHttpEndpoint;
+import com.griddynamics.jagger.invoker.v2.JHttpQuery;
+import com.griddynamics.jagger.invoker.v2.JHttpResponse;
+import com.griddynamics.jagger.test.jaas.util.TestContext;
+import com.griddynamics.jagger.test.jaas.util.entity.ExecutionEntity;
+import org.springframework.http.HttpMethod;
+
+public class ExecutionManipulateInvoker extends InvokerWithoutStatusCodeValidation {
```

Comment:
please add javadoc for every invoker, validator. it should be clear what is the goal of every class